docs/bash_to_xsh.rst

```diff
@@ -1,11 +1,7 @@
 Bash to Xonsh Translation Guide
 ================================
-As you have probably figured out by now, xonsh is not ``sh``-lang compliant.
-If your muscles have memorized all of the Bash prestidigitations, this page
-will help you put a finger on how to do the equivalent task in xonsh.
-
-For shell scripts, the recommended file extension is ``xsh``, and shebang
-line is ``#!/usr/bin/env xonsh``.
+As you have probably figured out by now, xonsh is not POSIX compliant.
+This page will help you put a finger on how to do the equivalent task in xonsh.
```

Comment:
Without the prior mention of Bash, the phrase "equivalent task" is lacking a referent. How about:

```suggestion
This page provides xonsh equivalents for common patterns in Bash.
```